pkg/gofr/container/container.go

```diff
@@ -117,6 +118,8 @@ func (c *Container) Create(conf config.Config) {
 
 		port, _ := strconv.Atoi(conf.Get("MQTT_PORT"))
 		order, _ := strconv.ParseBool(conf.GetOrDefault("MQTT_MESSAGE_ORDER", "false"))
+		maxReconnectInterval, _ := time.ParseDuration(conf.GetOrDefault("MQTT_MAX_RECONNECT_INTERVAL", "10m"))
+		keepAlive, _ := time.ParseDuration(conf.GetOrDefault("MQTT_KEEP_ALIVE", "30s"))
 
```

Comment:
it should be logged at debug level, that we are using these at default

Reply:
done